pkg/ingress/endpoint.go

```diff
@@ -181,8 +181,8 @@ func (c *endpointsController) onAdd(obj interface{}) {
 	if !c.controller.namespaceWatching(key) {
 		return
 	}
-	log.Debugw("endpoints add event arrived",
-		zap.Any("object", obj))
+	//log.Debugw("endpoints add event arrived",
```

Comment:
Why comment it?

Reply:
got error when logging.

Reply:
error?

Reply:
Ping @gxthrj .

Reply:
if log "object" with `zap.Any`, got error ` bad access: nil dereference`.
I changed logging field.